tests/muc/test_service.py

```diff
@@ -138,14 +138,30 @@ def test_init_full(self):
 
         self.assertEqual(
             occ.direct_jid,
-            TEST_ENTITY_JID
+            TEST_ENTITY_JID.bare()
         )
 
         self.assertEqual(
             occ.is_self,
             unittest.mock.sentinel.is_self,
         )
 
+    def test_init_full_jid_raises_ValueError(self):
+        with self.assertRaisesRegex(ValueError,
+                                    "^the jid argument must be a bare JID$"):
```

Comment:
I suggest to use ``r""`` instead of ``""`` for string literals which are used as regex – just so that somebody who modifies it down the road isn’t bitten by it.

Reply:
Good point.